modules/apps/site/site-admin-web/src/main/resources/META-INF/resources/js/SiteLanguageConfiguration.js

```diff
@@ -203,13 +203,13 @@ export default function SiteLanguageConfiguration({
 					/>
 
 					<ClayDualListBox
```

Comment:
Well, I'm not sure which team handles this part, just so we can tag them here to notify them. @julien do you know if it is a team from Spain?